serving-runtimes/vllm_runtime/vll-runtime-cpu.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+labels:
+  opendatahub.io/dashboard: "true"
+metadata:
+  annotations:
+    openshift.io/display-name: vLLM
+  name: vllm
+spec:
+  builtInAdapter:
+    modelLoadingTimeoutMillis: 90000
+  containers:
+    - args:
+        - --model
+        - /mnt/models/
+        - --download-dir
+        - /models-cache
+        - --port
+        - "8080"
+        - --max-model-len
```

Comment:
should we remove that so that it does not break when the model len does not fit with the specified one?

Reply:
Good point, I thought about it but think it's good to keep, it's not infrequent that it needs to be adjusted to make sure things fit within the memory.